frontend/src/constants/formats.js

```diff
@@ -3,3 +3,7 @@ export const dateFormat = 'DD/MM/YYYY';
 export const timeFormat = 'HH:mm';
 
 export const hourFormat = 'h';
+
+export const MAX_LENGTH = 50;
+
+export const HTTP = 'http';
```

Comment:
There are date constants, move MAX_LENGTH, 'http' to other file

Reply:
done